src/migrations/023.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE division
+  ADD COLUMN draft_sheet varchar(100);
```

Comment:
Let's make this `draft_sheet_url` so it's a bit more clear - at first glance I honestly wasn't sure what I was looking at. Additionally, 100 characters is cutting it close for Google Sheets links - let's set that to at least 300 to be safe.